common/types.go

```diff
@@ -1077,7 +598,7 @@ func (s *SubmitBlockRequestV2Optimistic) SizeSSZ() (size int) {
 
 	// Field (1) 'ExecutionPayloadHeader'
 	if s.ExecutionPayloadHeader == nil {
-		s.ExecutionPayloadHeader = new(consensuscapella.ExecutionPayloadHeader)
+		s.ExecutionPayloadHeader = new(capella.ExecutionPayloadHeader)
```

Comment:
to be honest I'm not too familiar with the optimistic code path and whether @michaelneuder would like to update this for deneb?
Otherwise I can try to add changes for the upgrade as a follow up PR